pkg/api/org_users_test.go

```diff
@@ -24,10 +26,24 @@ func setUpGetOrgUsersHandler() {
 	})
 }
 
+func setUpGetOrgUsersDB(t *testing.T, sqlStore *sqlstore.SQLStore) {
+	setting.AutoAssignOrg = true
+	setting.AutoAssignOrgId = 1
```

Comment:
Changes to state need to be cleaned up after the test is over, otherwise other tests get affected, which is a no-no:
```suggestion
	origAutoAssignOrg := setting.AutoAssignOrg
	origAutoAssignOrgId := setting.AutoAssignOrgId
	t.Cleanup(func() {
		setting.AutoAssignOrg = origAutoAssignOrg
		setting.AutoAssignOrgId = origAutoAssignOrgId
	})
	setting.AutoAssignOrg = true
	setting.AutoAssignOrgId = 1
```